addons/elasticsearch/elasticsearch.yaml

```
@@ -10,9 +10,11 @@ metadata:
     # on the cluster, this hack will trigger re-queue on Addons until one exists.
     kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
   annotations:
-    catalog.kubeaddons.mesosphere.io/addon-revision: "6.8.2-8"
-    appversion.kubeaddons.mesosphere.io/elasticsearch: "6.8.2"
-    values.chart.helm.kubeaddons.mesosphere.io/elasticsearch: "https://raw.githubusercontent.com/mesosphere/charts/73fba37/stable/elasticsearch/values.yaml"
+    catalog.kubeaddons.mesosphere.io/addon-revision: "7.10.1-1"
+    appversion.kubeaddons.mesosphere.io/elasticsearch: "7.10.1"
+    values.chart.helm.kubeaddons.mesosphere.io/elasticsearch: "https://raw.githubusercontent.com/mesosphere/charts/master/stable/elasticsearch/values.yaml"
```

```suggestion
    values.chart.helm.kubeaddons.mesosphere.io/elasticsearch: "https://raw.githubusercontent.com/mesosphere/charts/87308490091d58268711646ca3fd44cbc55f47bc/stable/elasticsearch/values.yaml"
```
I think better to pin the values to sha instead of master

---

yep, was about to do that :)